test.disabled/ejabberd_tests/tests/mam_SUITE.erl

```diff
@@ -636,38 +636,44 @@ init_modules(BackendType, muc_light, Config) ->
 init_modules(cassandra, muc_all, Config) ->
     init_module(host(), mod_mam_muc_cassandra_arch, []),
     init_module(host(), mod_mam_muc, [{host, muc_domain(Config)}, add_archived_element]),
+    true = rpc_apply(code, delete, [mod_mam_params]);
```

Comment:
If you're happy and you know it, syntax error.
If you're happy and you know it, syntax error.
If you're happy and you know it and you really want to show it, if you're happy and you know it, syntax error.

Reply:
WIP label, @fenek!

Reply:
Oh come on, why so serious? :)